spring-xd-lang-detector/src/main/resources/config/spring-module.properties

```diff
@@ -0,0 +1,27 @@
+options.textInput.description = The name of the property that contains the input text.
+options.textInput.default = text
+options.textInput.type = java.lang.String
+
+options.langOutput.description = The name of the output property the detected language is written to.
```

Comment:
Sure, there are a lot in XD itself. Or https://github.com/ericbottard/spring-xd-modules/blob/XD-2788/throughput/src/main/java/org/springframework/xd/throughput/ThroughputOptionsMetadata.java